src/components/DisplayInput.js

```diff
@@ -0,0 +1,145 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+const DisplayInput = React.createClass({
+  propTypes: {
+    hint: React.PropTypes.string,
+    label: React.PropTypes.string,
+    placeholder: React.PropTypes.string,
+    primaryColor: React.PropTypes.string,
+    showHint: React.PropTypes.bool,
+    status: React.PropTypes.shape({
+      type: React.PropTypes.string,
+      message: React.PropTypes.string
+    }),
+    valid: React.PropTypes.bool
+  },
+
+  getDefaultProps () {
+    return {
+      primaryColor: StyleConstants.Colors.PRIMARY,
+      valid: true
+    };
+  },
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <div style={Object.assign({}, styles.wrapper)}>
```

Comment:
No need for the `Object.assign` if your not merging two or more style objects together.